### PR TITLE
Workshop 3

### PR DESCRIPTION
**Pull request by peterastrong**

### CODE REVIEW

File: app/controllers/registrants_controller.rb [hunk 1]
```diff
@@ -1,34 +1,20 @@
...
 
   def create
-    @registrant = Registrant.new(first_name: params[:first_name], last_name: params[:last_name], affiliation: params[:affiliation], occupation: params[:occupation],  email: params[:email])
-    @registrant.save 
-  end
-
-  def show
-    @registrant = Registrant.find_by(id: params[:id])
-    render 'show.html.erb'    
+    registrant = Registrant.new(first_name: params[:first_name], last_name: params[:last_name], affiliation: params[:affiliation], occupation: params[:occupation],  email: params[:email], phone: params[:phone])
```
Comment (trevordjones) [inline app/controllers/registrants_controller.rb]: Look at `params.require` from other controllers

Goal is to `Registrant.new(registrant_params)`

File: app/views/workshops/new.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 </div>
 <div>
  <%= label_tag :date %>
- <%= text_field_tag :date  %>
+ <%= text_field_tag :date, @workshop.date   %>
```
Comment (trevordjones) [inline app/views/workshops/new.html.erb]: REmove this guy

File: db/migrate/20170518223205_create_registrant_workshops.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateRegistrantWorkshops < ActiveRecord::Migration[5.0]
+  def change
+    create_table :registrant_workshops do |t|
+      t.string :registrant_id
+      t.string :workshop_id
```
Comment (trevordjones) [inline db/migrate/20170518223205_create_registrant_workshops.rb]: Make these integers

**Comment by peterastrong** [comment]
Made changes

**Pull request merged**